From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 3 Sep 2020 15:12:50 +0100
Subject: dt: Remove duplicate assignment for i2c0 pinctrl config

The include file bcm283x-rpi-i2c0mux_0_XX.dtsi was setting
pinctrl-0 on i2c0mux, as were the individual platform DT files.

Remove this duplication and rely on the include file.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2708-rpi-b-plus.dts   | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-b.dts        | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-cm.dts       | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-zero-w.dts   | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-zero.dts     | 4 ----
 arch/arm/boot/dts/bcm2709-rpi-2-b.dts      | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-2-b.dts      | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-3-b.dts      | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-cm3.dts      | 4 ----
 10 files changed, 40 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
index 0e137d85897b..98581eec4bdc 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-b.dts b/arch/arm/boot/dts/bcm2708-rpi-b.dts
index 47c093364683..305f1dbde60f 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-cm.dts b/arch/arm/boot/dts/bcm2708-rpi-cm.dts
index 5dcdf8888ec1..93062c4ffad2 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-cm.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-cm.dts
@@ -72,10 +72,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
index 0060c9bfe2e6..f1bbed9fbf73 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
@@ -122,10 +122,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-zero.dts b/arch/arm/boot/dts/bcm2708-rpi-zero.dts
index 5655eb4769ea..e7578788b839 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-zero.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero.dts
@@ -76,10 +76,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
index 77678a3d7ef9..6b2e3c291d72 100644
--- a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-2-b.dts b/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
index dd7e1897ea45..49cfda63606e 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
index 0d6a47118b83..4e4e47100831 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
@@ -132,10 +132,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
index 7dc3ef47e78f..8989c00b03e5 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -147,10 +147,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
index 88f7fe53b97f..f972979281f5 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
@@ -92,10 +92,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
